candi/openapi/doc-ru-cluster_configuration.yaml

```
@@ -16,11 +16,13 @@ apiVersions:
       apiVersion:
         description: Используемая версия API Deckhouse.
       clusterType:
+        x-unsafe: true
```

```suggestion
      clusterType:
```

It is not necessary to fix doc-ru-* resources - they are only for translated description fields.